app/components/organization-team/add-team-member/index.scss

```diff
@@ -0,0 +1,25 @@
+.emtpy-container {
```

Comment:
Some of the declarations here can be replaced with our utility classes. Don't you think?

Reply:
yeah but I would still require the class so keeping everything together